src/realm/group_writer.hpp

```diff
@@ -122,30 +153,16 @@ class GroupWriter : public _impl::ArrayWriterBase {
     size_t m_window_alignment;
     size_t m_free_space_size = 0;
     size_t m_locked_space_size = 0;
+    size_t m_evacuation_limit;
+    int m_backoff;
```

Comment:
why not make it a size_t, or even a int64_t which is the stored format?

Reply:
Now I went in the other direction and made it a `bool`. If I at a later point in time want it to be a counter, I will make it an int64_t.